components/blocks/cards/SignUpCard/SignUpCard.vue

```diff
@@ -53,42 +61,6 @@ const passwordInputValid = ref(true)
 const passwordConfirmValid = ref(true)
 const usernameValid = ref(true)
 
-// Regex(s) //
```

Comment:
ya i should've realized that it'd be better to move these out of this lol

Reply:
Only really useful now that we're doing this check in more places, but yeah lol